vowpalwabbit/parse_example_json.h

```diff
@@ -314,11 +314,8 @@ struct LabelState : BaseState<audit>
 
   BaseState<audit>* StartObject(Context<audit>& ctx) { return ctx.label_object_state.StartObject(ctx); }
 
-  BaseState<audit>* String(Context<audit>& ctx, const char* str, rapidjson::SizeType /* len */, bool copy)
```

Comment:
I think this copy is part of the API required by the rapidjson parser: http://rapidjson.org/md_doc_sax.html#CustomDataStructure. 

I am guessing this is what is causing the parser failures in the unit-tests.

Reply:
Looks like you were way ahead of me.